Document how to make namespace wildcard intentions. (#10724)
hashicorp/consul: website/content/docs/connect/intentions.mdx

> An intention source or destination may also be the special wildcard

Reviewer: ```suggestion
You can use the `*` wildcard when defining an intention source or destination. The wildcard matches _any_ value and can serve as a "catch-all" entry for intentions that should have a wide scope.     
```

This suggestion about using language more consistent with our style guide and to define the purpose of a "catch-all" entry. Please let me know if it is inaccurate and I can try providing a better suggestion.

> value `*`. This matches _any_ value and is used as a catch-all.

Reviewer: ```suggestion

```

> Wildcards can be used for service name matching. In Consul <EnterpriseAlert inline />

Reviewer 2: ```suggestion
Wildcards can be used for service name matching. In Consul Enterprise
```
We've typically only used these labels on section headings, or right after variables names in the API/CLI config options, but AFAIK not in the middle of a sentence.

Reviewer: Do we support just the any-character wildcard? Would there every need be a need to specify something like `**/*-service` or `service_?` ?

Reviewer: ```suggestion
You can use a wildcard to match service names. If you are using Consul Enterprise, you can also use a wildcard to match a namespace.
```

> they can also be used for Namespace matching.

Reviewer: ```suggestion

```

]
```

> This <EnterpriseAlert inline /> example grants access to Prometheus 
> access to any service in any namespace.

Reviewer 2: ```suggestion
<EnterpriseAlert inline /> This example grants access to Prometheus
from any service in any namespace.
```
